python/lsst/ts/m2gui/controltab/tab_config_view.py

```diff
@@ -33,7 +37,319 @@ class TabConfigView(TabDefault):
         Table's title.
     model : `Model`
         Model class.
+
+    Attributes
+    ----------
+    model : `Model`
+        Model class.
     """
 
     def __init__(self, title, model):
         super().__init__(title, model)
+
+        # Configuration parameters
+        self._config_parameters = {
+            "file_configuration": create_label(),
+            "file_version": create_label(),
+            "file_control_parameters": create_label(),
+            "file_lut_parameters": create_label(),
+            "power_warning_motor": create_label(),
+            "power_fault_motor": create_label(),
+            "power_threshold_motor": create_label(),
+            "power_warning_communication": create_label(),
+            "power_fault_communication": create_label(),
+            "power_threshold_communication": create_label(),
+            "in_position_axial": create_label(),
+            "in_position_tangent": create_label(),
+            "in_position_sample": create_label(),
+            "timeout_sal": create_label(),
+            "timeout_crio": create_label(),
+            "timeout_ilc": create_label(),
+            "misc_range_angle": create_label(),
+            "misc_diff_enabled": create_label(),
+            "misc_range_temperature": create_label(),
+        }
+
+        self._set_widget_and_layout()
+
+        self._set_signal_config(self.model.signal_config)
+
+    def _set_widget_and_layout(self):
+        """Set the widget and layout."""
+
+        widget = self.widget()
+        widget.setLayout(self._create_layout())
+        self.setWidget(self.set_widget_scrollable(widget, is_resizable=True))
+
+    def _create_layout(self):
+        """Create the layout.
+
+        Returns
+        -------
+        layout : `PySide2.QtWidgets.QVBoxLayout`
+            Layout.
+        """
+
+        layout = QVBoxLayout()
+        layout.addWidget(self._create_group_file_version())
+        layout.addWidget(self._create_group_power_motor())
+        layout.addWidget(self._create_group_power_communication())
+        layout.addWidget(self._create_group_in_position_flag())
+        layout.addWidget(self._create_group_communication_timeout())
+        layout.addWidget(self._create_group_misc_inclinometer())
+        layout.addWidget(self._create_group_misc_temperature())
+
+        return layout
+
+    def _create_group_file_version(self):
+        """Create the group of files and version.
+
+        Returns
+        -------
+        group : `PySide2.QtWidgets.QGroupBox`
+            Group.
+        """
+
+        group = QGroupBox("File Paths and Version")
+
+        layout = QFormLayout()
+        layout.addRow(
+            "File of Configuration: ", self._config_parameters["file_configuration"]
+        )
+        layout.addRow(
+            "Configuration Version: ", self._config_parameters["file_version"]
```

Comment:
There isnt'a a need to add " " (space) at the end.

Reply:
Fixed in the commit: [884ad1e](https://github.com/lsst-ts/ts_m2gui/pull/6/commits/884ad1e142bee440958cb41fc57aff9ed6022243). Thanks!